test/e2e/testing-manifests/sample-device-plugin/sample-device-plugin-control-registration.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: sample-device-plugin-beta
+  namespace: kube-system
+  labels:
+    k8s-app: sample-device-plugin
+spec:
+  selector:
+    matchLabels:
+      k8s-app: sample-device-plugin
+  template:
+    metadata:
+      labels:
+        k8s-app: sample-device-plugin
+      annotations:
+    spec:
+      priorityClassName: system-node-critical
+      tolerations:
+      - operator: "Exists"
+        effect: "NoExecute"
+      - operator: "Exists"
+        effect: "NoSchedule"
+      volumes:
+      - name: device-plugin
+        hostPath:
+          path: /var/lib/kubelet/device-plugins
+      - name: plugins-registry-probe-mode
+        hostPath:
+          path: /var/lib/kubelet/plugins_registry
+      - name: dev
+        hostPath:
+          path: /dev
+      containers:
+      - image: registry.k8s.io/e2e-test-images/sample-device-plugin:1.3
```

Comment:
I think we will need to bump the device plugin version, which can complicate the matters, but let's check.

Reply:
I am not very familiar with the sample-device-plugin image build process, will look into that. We would have to make sure that the image gets built with right tag is provided here.

Reply:
ok, the trick here is that tests will consume the existing image already pushed in the registry. It's possible we will need to split the related commits to its own PR, then once merged, push a new sample-device-plugin image and finally consume it in the e2e tests here.
I'm not sure we can just bundle the changes and consume them in the same PR. Let's check.

Reply:
Yes, that could be the case here. I will look into how this was done previously.

Reply:
I will bump the sample device plugin version, point to that in the manifests and let's see how the CI reacts. That should give us an indication if the changes related to sample device plugin should be moved to a separate PR.